sandman/source/schedule.cpp

```diff
@@ -221,8 +244,7 @@ static void ScheduleLogLoaded()
 void ScheduleInitialize()
 {	
 	s_ScheduleIndex = UINT_MAX;
-	
-	s_ScheduleEvents.clear();
+	delete(s_Schedule);
```

Comment:
I'm pretty sure the usual syntax is just delete s_Schedule. I didn't think that this would compile.